event-streams/grafana-dashboards/README.md

```diff
@@ -0,0 +1,7 @@
+# Grafana-Dashboards Samples
+This directory contains files for configuring Event Streams cluster health with Prometheus and Grafana
```

Comment:
```suggestion
This directory contains sample configuration files to help you set up cluster health monitoring with Prometheus and Grafana.
```

Reply:
Is there a reason we provide an explanation and links into the samples within the folder here, but we don't do it for others the same way (for example, `kafka-connect` or `kafka-user` dirs, or any other which has more than one sample).

Reply:
@gabortpubs I believe we don't provide much information about the grafana configuration files in our official docs but for the rest of them we do have and  For `kafka-user` dirs there are a lot of files and I dint want to overdo a lot of information into the Readme.

But for a matter of consistency, we can remove the explanation and the links to the configuration file in the grafana directory.